src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -77,4 +77,14 @@
     public ResponseEntity<Flux<UUID>> getModificationGroups() {
         return ResponseEntity.ok().body(networkModificationService.getModificationGroups());
     }
+
+    @PutMapping(value = "/networks/{networkUuid}/lines/{lineId}/state", consumes = MediaType.TEXT_PLAIN_VALUE)
```

Comment:
Status?

Reply:
I prefer the state term because it does not only change the status (i.e. plenned outage, forced outage...) but also the state of the line (connected, disconnected...)

Reply:
API modified to use "status"